deploy/ansible/host_vars/vagrant

```diff
@@ -18,9 +18,9 @@ hostname: ansible-vagrant
 site_fqdn: "{{project_name}}.loc"
 
 # Should the nginx server use HTTPS instead of HTTP?
-ssl: false
+# ssl: false
 
 # If ssl is enabled, these cert/key files will be used by nginx. Self-signed
 # cert/key files will be auto-generated if they don't already exist.
-ssl_cert_path: /etc/ssl/server.cert
-ssl_key_path: /etc/ssl/server.key
+# ssl_cert_path: /etc/ssl/server.cert
+# ssl_key_path: /etc/ssl/server.key
```

Comment:
We can remove all this

Reply:
removed in most recent commit -- thanks!